tests/vlan/test_portchannel_tag.py

```diff
@@ -0,0 +1,330 @@
+
+import pytest
+import ptf.packet as scapy
+import ptf.testutils as testutils
+from ptf.mask import Mask
+from collections import defaultdict
+
+import json
+import itertools
+import logging
+
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py       # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses        # lgtm[py/unused-import]
+
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor  # lgtm[py/unused-import]
+
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('topo_t0-56-po2vlan')
+]
+
+
+@pytest.fixture(scope="module")
+def cfg_facts(duthosts, rand_one_dut_hostname):
+    duthost = duthosts[rand_one_dut_hostname]
+    return duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
+
+
+@pytest.fixture(scope="module")
+def vlan_ports_list(rand_selected_dut, tbinfo, cfg_facts):
+    mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
+    vlan_ports_list = []
+    config_ports = {k: v for k,v in cfg_facts['PORT'].items() if v.get('admin_status', 'down') == 'up'}
+    config_portchannels = cfg_facts.get('PORTCHANNEL', {})
+    config_port_indices = {k: v for k, v in mg_facts['minigraph_ptf_indices'].items() if k in config_ports}
+    ptf_ports_available_in_topo = {port_index: 'eth{}'.format(port_index) for port_index in config_port_indices.values()}
+    config_port_channel_members = [port_channel[1]['members'] for port_channel in config_portchannels.items()]
```

Comment:
>port_channel[1] [](http://example.com/codeflow?start=35&length=15)

What is the data in `port_channel`? Do you need `[1]` ?

Reply:
This line is from test_vlan.py.
items() returns a tuple, so [1] is used to get the second member.
This is redundant, and I will fix it